docs/examples/grpc/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM golang as build
```

Comment:
We should probably specify a go version here :)

Reply:
Similar to above comment, one one hand I see advantage of hardcoding versions. And on the other hand I don't know if we want to create reason for outdating images/builds and requiring regular PRs for updates/fixes. Looking forward to your advise @rikatz

Reply:
for example, I want to work on the doc for the prometheus monitoring, but our hardcoded version of prometheus in the build system is 2.3 and the current release of prometheus is 2.28 .

Reply:
So we can in this case put the go version this image is being built, also so this wont break in a future and keep updating as necessary.

Reply:
changed FROM: golang to FROM: golang:alpine. This currently default to 1.16.5 and in future will automatically pickup updated go versions so we don't need to risk outdating